### PR TITLE
SBERDOMA-453 fix empty line and wrong address in ticket change messages

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**

### CODE REVIEW

File: apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx [hunk 1]
```diff
@@ -36,25 +46,56 @@ export const TicketChange: React.FC<ITicketChangeProps> = ({ ticketChange }) =>
...
+
+    const getChangeMessages = (baseId: string) => {
+        return {
+            add: intl.formatMessage({ id: `${baseId}.add` }),
+            change: intl.formatMessage({ id: `${baseId}.change` }),
+            remove: intl.formatMessage({ id: `${baseId}.remove` }),
+        }
+    }
+
+    const fields: ITicketChangeField[] = [
```
Comment (pahaz) [inline apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx:58]: we have a style convention about `intl` it should be at the top of the component.

File: apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx [hunk 2]
```diff
@@ -132,15 +161,38 @@ const useChangedFieldMessagesOf = (ticketChange) => {
...
+                    {aroundFrom[1]}
+                </>
+            )
         }
     }
 
     // Omit what was not changed
-    const changedFields = fields.filter(([field]) => (
-        ticketChange[`${field}From`] !== ticketChange[`${field}To`]
+    const changedFields = fields.filter(({ title }) => (
```
Comment (pahaz) [inline apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx:179]: please don't do any redundant changes it's hard to review

File: apps/condo/domains/contact/schema/Contact.test.js [hunk 3]
```diff
@@ -249,7 +249,7 @@ describe('Contact', () => {
             const adminClient = await makeLoggedInAdminClient()
             const [obj] = await createTestContact(adminClient, userClient.organization, userClient.property)
             const objs = await Contact.getAll(userClient, {}, { sortBy: ['updatedAt_DESC'] })
-            expect(objs.length).toEqual(1)
+            expect(objs).toHaveLength(1)
```
Comment (toplenboren) [inline apps/condo/domains/contact/schema/Contact.test.js:252]: 👍

**Review by toplenboren — APPROVED** [review]

**Comment by AntonAL** [comment]
Address change is not being displayed.

image

File: apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx [hunk 4]
```diff
@@ -36,25 +46,82 @@ export const TicketChange: React.FC<ITicketChangeProps> = ({ ticketChange }) =>
...
+        {
+            title: 'details',
+            messages: {
+                change: DetailsChangeMessage,
+            },
+        },
+        {
+            title: 'clientName',
+            messages: {
+                change: ClientNameChangeMessage,
```
Comment (pahaz) [inline apps/condo/domains/ticket/components/TicketChanges/TicketChange.tsx:78]: Is it working if we add a client name?

**Pull request merged**